paddle/fluid/framework/op_desc.h

```diff
@@ -80,6 +80,15 @@ class OpDesc {
 
   Attribute GetAttr(const std::string &name) const;
 
+  template <typename T>
+  T GetAttrIfExists(const std::string &name) const {
+    T result;
```

Comment:
This cause undefined behavior for built-in data types.

Reply:
Great observation, thanks!